src/cluster/__tests__/connectionBuilder.spec.js

```diff
@@ -43,11 +43,13 @@ describe('Cluster > ConnectionBuilder', () => {
   })
 
   test('when called without host and port iterates throught the seed brokers', () => {
-    expect(builder.build()).toEqual(expect.objectContaining({ host: 'host.test', port: 7777 }))
-    expect(builder.build()).toEqual(expect.objectContaining({ host: 'host2.test', port: 7778 }))
-    expect(builder.build()).toEqual(expect.objectContaining({ host: 'host3.test', port: 7779 }))
-    expect(builder.build()).toEqual(expect.objectContaining({ host: 'host.test', port: 7777 }))
-    expect(builder.build()).toEqual(expect.objectContaining({ host: 'host2.test', port: 7778 }))
+    const connections = Array(brokers.length)
+      .fill()
+      .map(() => {
+        const { host, port } = builder.build()
+        return `${host}:${port}`
+      })
+    expect(connections).toIncludeSameMembers(brokers)
```

Comment:
This validates that all of the brokers are returned, just not in the same order.